.gitignore

```diff
@@ -150,3 +150,4 @@ flatbuffers.pc
 **/latex/**
 # https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_BASE_DIR
 _deps/
+tests/ts/arrays_test_complex/arrays_test_complex.bfbs
```

Comment:
this probably shouldn't be in here

Reply:
@dbaileychess should I add the bfbs to the repo? It is re-generated everytime by [this](https://github.com/zdrealityhub/flatbuffers/blob/c716af815b3b6b5dd3017090a5bdd0ca434d4470/tests/ts/TypeScriptTest.py#L104) for testing.

Reply:
I vote to add, for consistency (we have all other test generated output in repo to track changes to it).

Reply:
Makes sense then. I added it. 1867a9c807548ed580de35468f23b27de94e6cdd